.github/workflows/delete-merged-branches.yml

```diff
@@ -137,8 +137,7 @@ jobs:
             merged_prs=$(echo "$pr_response" | \
               jq '[.[] | select(.merged_at != null)] | length')
             closed_prs=$(echo "$pr_response" | \
-              jq '[.[] | select(.state == "closed" and .merged_at == null)] \
-              | length')
+              jq '[.[] | select(.state == "closed" and .merged_at == null)] | length')
```

Comment:
Change to not split across multiple lines and update all usages of jq to not  split.

That is change to the following to avoid potential line splitting issues
`closed_prs=$(echo "$pr_response" | jq '[.[] | select(.state == "closed" and .merged_at == null)] | length')`